REFERENCE.md

```diff
@@ -17,6 +17,19 @@
 * `chrony::params`: chrony class parameters
 * `chrony::service`: Manages the chrony service
 
+### Functions
+
+#### Public Functions
```

Comment:
Perhaps worth a bug report to puppet-strings that it shouldn't output this header if there are no public functions.

Reply:
I've opened https://github.com/puppetlabs/puppet-strings/issues/267